src/panels/lovelace/cards/hui-picture-glance-card.ts

```diff
@@ -332,11 +332,11 @@ class HuiPictureGlanceCard extends LitElement implements LovelaceCard {
       ha-icon-button {
         --mdc-icon-button-size: 40px;
         --disabled-text-color: currentColor;
-        color: #a9a9a9;
+        color: var(--ha-pic-icon-button-color, #a9a9a9);
       }
 
       ha-icon-button.state-on {
-        color: white;
+        color: var(--ha-pic-icon-button-on-color, white);
```

Comment:
These 2 are more related to the glance card than the picture card I think?

Reply:
Sorry, I don't understand what you mean.  They are only in the glance card file.

Reply:
But the naming of the vars is for a picture card

Reply:
Ah!  I did that because:

* Glance is a sub-type of picture
* The other new variables are --ha-picture-*, so I was keeping the naming consistent
* If another card type is introduced in the future that doesn't have glance in the name that could re-use them, it seems like a more generic name would be better.

So I see three choices:
1. --ha-picture-icon-button-color
2. --ha-picture-glance-icon-button-color
3. --ha-glance-icon-button-color

In my mind (1) makes the most sense, but I am happy to defer to what you think would be best - just let me know!